web-bundle/src/main/java/no/ecc/vectortile/VectorTile.java

```diff
@@ -0,0 +1,5735 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Instead of including this large file we could also just include the much smaller vector_tile.proto file and generate this using the protoc Maven plugin. But unfortunately this does not seem to be recognized by IntelliJ and one has to run `mvn compile` manually first?!